ibft/README.md

```diff
@@ -0,0 +1,87 @@
+# Istanbul Byzantine Fault Tolerance (IBFT)
```

Comment:
@vinipx this is a good start, I'd maybe shape it to be more eth2 secret-shared-validator focus. 
iBFT can be a general purpose consensus protocol but here our usage is more specific.

Reply:
Agree - I'd rather see this file content linked to the main ssv-ibft readme. 
TODO: 
1. Rename README.md -> ibft-explained.md
2. At README.me level (ibft package) elaborate IBFT briefing explanation linking to ibft-explained.md for further background refence

Reply:
not sure I understand. You want to have 2 readme?

Reply:
I'm afraid this might get too long file content. Therefore, maybe better to have a **Readme** covering SSV IBFT specifics and a link referencing to this one (IBFT explained) in a separate md file. 

What your thoughts?